client/tiny.js

```diff
@@ -85,27 +87,53 @@ function removeUser(session, userId) {
   }
 }
 
+let messages = [];
+let firstMessageTime;
+function storeMessage(ev) {
+  if (!firstMessageTime) {
+    firstMessageTime = performance.now();
+  }
+  messages.push({
+    time: performance.now() - firstMessageTime,
+    message: JSON.parse(ev.data)
+  });
+}
+
+document.getElementById("saveButton").addEventListener("click", function saveToMessagesFile() {
+  const file = new File([JSON.stringify(messages)], "messages.json", {type: "text/json"});
+  saveAs(file);
+});
+
+document.getElementById("clearButton").addEventListener("click", function clearMessages() {
+  messages = [];
+});
```

Comment:
Sure. Right now I don't have a principled distinction between tiny and squawkers -- it's just separate because I wanted to write one thing, and then later I found myself wanting to write a different thing without figuring out whether they should both be the same thing. So do whatever seems reasonable as far as putting stuff in them. Maybe we will make them the same soon.